index.html

```diff
@@ -10,6 +10,7 @@
     <link href='https://fonts.googleapis.com/css?family=Inter' rel='stylesheet'>
     <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet'>
     <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+    <link href='https://fonts.googleapis.com/css?family=Meera Inimai' rel='stylesheet'>
     <title>Document</title>
 </head>
```

Comment:
- [ ] Your HTML file and `<link>` tag to connect your styles sheet is not working correctly 🛠 and that is blocking your styles to be displayed. Usually, when your files are in the same directory you can only specify the file name like this: `<link rel="stylesheet" href="style.css">` and your styles shall be displayed 🎯. Please fix this 👇🏽

![image](https://user-images.githubusercontent.com/43172164/205328202-68f246e7-b6a5-4418-9162-dbd6f0c7980c.png)